features/admin.server-configurations.v1/constants/index.ts

```diff
@@ -19,3 +19,4 @@
 export * from "./server-configurations-constants";
 export * from "./governance-connector-constants";
 export * from "./password-recovery-constants";
+export * from "./username-recovery-constants";
```

Comment:
Let's not use index files for exports, instead import directly from the constant file

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7004/commits/59d040047dd8b21efd511e781c167e8ebd65b64f